TradeItIosTicketSDK2/TradeItLinkedBrokerCache.swift

```diff
@@ -1,62 +1,51 @@
 class TradeItLinkedBrokerCache {
-    typealias UserId = String
-    typealias SerializedLinkedBroker = [String: Any]
-    typealias SerializedLinkedBrokers = [UserId: SerializedLinkedBroker]
-    typealias SerializedLinkedBrokerAccount = [String: Any]
-
-    private let LINKED_BROKER_CACHE_KEY = "LINKED_BROKER_CACHE"
-
-    private let ACCOUNTS_KEY = "ACCOUNTS"
-    private let ACCOUNTS_LAST_UPDATED_KEY = "ACCOUNTS_LAST_UPDATED"
-    private let ACCOUNT_NAME_KEY = "ACCOUNT_NAME"
-    private let ACCOUNT_NUMBER_KEY = "ACCOUNT_NUMBER"
-    private let ACCOUNT_INDEX_KEY = "ACCOUNT_INDEX"
-    private let ACCOUNT_BASE_CURRENCY_KEY = "ACCOUNT_BASE_CURRENCY"
-    private let ACCOUNT_ENABLED_KEY = "ACCOUNT_ENABLED"
-    private let ACCOUNTS_LINK_DELAY_ERROR_KEY = "ACCOUNTS_LINK_DELAY_ERROR"
-    private let BALANCE_LAST_UPDATED_KEY = "BALANCE_LAST_UPDATED"
-    private let BALANCE_BUYING_POWER_KEY = "BALANCE_BUYING_POWER"
-    private let FX_BALANCE_BUYING_POWER_KEY = "FX_BALANCE_BUYING_POWER"
-
-    private let ACCOUNT_ENABLED_VALUE = "ENABLED"
-    private let ACCOUNT_DISABLED_VALUE = "DISABLED"
-
-    internal static var _userDefaults = UserDefaults(suiteName: "it.trade")!
-    internal var userDefaults: UserDefaults {
-        get {
-            return TradeItLinkedBrokerCache._userDefaults
-        }
-    }
 
+    private let LINKED_BROKER_CACHE_KEY = "TRADE_IT_LINKED_BROKER_CACHE_"
+  
     func cache(linkedBroker: TradeItLinkedBroker?) {
         guard let linkedBroker = linkedBroker else { return }
-
-        var linkedBrokerCache = userDefaults.dictionary(forKey: LINKED_BROKER_CACHE_KEY) as? SerializedLinkedBrokers ?? SerializedLinkedBrokers()
-
-        let serializedLinkedBroker = serialize(linkedBroker: linkedBroker)
-
-        linkedBrokerCache[linkedBroker.linkedLogin.userId] = serializedLinkedBroker
-
-        self.userDefaults.set(linkedBrokerCache, forKey: LINKED_BROKER_CACHE_KEY)
+        let key = LINKED_BROKER_CACHE_KEY+linkedBroker.linkedLogin.userId
```

Comment:
`linkedBroker.linkedLogin.userId` can be changed to `linkedBroker.userId`. There is a convenience computed property getter on `TradeItLinkedBroker`.